CSharp/Library/Microsoft.Bot.Builder/Dialogs/IDialogContext.cs

```diff
@@ -141,6 +141,7 @@ public static partial class Extensions
         {
             var message = botToUser.MakeMessage();
             message.Text = text;
+            message.InputHint = InputHints.ExpectingInput;

```

Comment:
AutoInputHint_BotToUser will automatically apply InputHints to messages that don't have InputHint: https://github.com/Microsoft/BotBuilder/blob/master/CSharp/Library/Microsoft.Bot.Builder/ConnectorEx/BotToUser.cs#L182

It is wrong to set the InputHint to expecting for the cases that user might use the following PostAysnc: https://github.com/Microsoft/BotBuilder/blob/master/CSharp/Library/Microsoft.Bot.Builder/Dialogs/IDialogContext.cs#L142 to send multiple messages to channel.